js/factory/errors.js

```diff
@@ -0,0 +1,13 @@
+
+app.factory('Errors', function($timeout) {
```

Comment:
naming convention - conform to ErrorsFactory? (or change others to not have 'factory' in them)